coala.css

```diff
@@ -776,6 +776,11 @@ a.chip i {
   cursor: pointer;
 }
 
+.coala-online .add-bear {
```

Comment:
I think having a rule just for a single type of button is unnecessary. Please consider a more general way to solve this problem.

Reply:
@ksdme I have tried most of the ways. Since adding inline styles is not allowed, adding a class to CoalaCSS is the only option. I even moved out the "Add Bear" text out of the icon tag but that does not change its font to Roboto i.e additional style has to applied to it, which only leaves the option of adding a class. Besides every item on coala-online that uses a mix of Roboto and Mono font has a class defined for it in coalaCSS so I think "Add Bear" text should have it too. At present this is the best approach I can think of, if you have an alternative better approach could you please suggest it to me.

Reply:
https://github.com/coala/landing-frontend/pull/284#issuecomment-414041767; seemed to work instantly for me.

Reply:
Yup it works. Thanks :)